test/mocks/index.js

```diff
@@ -7,79 +7,91 @@ export const user = {
   googleId: '123456789',
   facebookId: '123456789',
 };
-export const user4 = {
-  UserName: 'ke',
-  Email: 'itiswasp@gmail.com',
-  Password: 'Andelahfghfh',
-  FirstName: 'JohnS',
-  LastName: 'DoeS',
+
+export const invalidPasswordUser = {
+  userName: 'kevi',
+  email: 'didas@andela.com',
+  password: 'Andelahfghfh',
+  firstName: 'JohnS',
+  lastName: 'DoeS',
 };
-export const user5 = {
-  UserName: 'kevi',
-  Email: 'didas@andela.com',
-  Password: 'Andela@2022',
-  FirstName: 'JohnS',
-  LastName: 'DoeS',
+
+export const invalidUsernameUser = {
+  userName: 'ke',
+  email: 'itiswasp@gmail.com',
+  password: 'Andelahfghfh',
+  firstName: 'JohnS',
+  lastName: 'DoeS',
 };
-export const user6 = {
-  UserName: 'kevi',
-  Email: 'itiswasp@gmail.com',
-  Password: 'Andelahfghfh',
-  FirstName: 'Jo',
-  LastName: 'DoeS',
+
+export const invalidEmailUser = {
+  userName: 'kevi',
+  email: 'didas@andela',
+  password: 'Andela@2022',
+  firstName: 'JohnS',
+  lastName: 'DoeS',
 };
-export const user7 = {
-  UserName: 'kevi',
-  Email: 'itiswasp@gmail.com',
-  Password: 'Andelahfghfh',
-  FirstName: 'JohnS',
-  LastName: 'DS',
+
+export const invalidFirstNameUser = {
+  userName: 'kevi',
+  email: 'itiswasp@gmail.com',
+  password: 'Andelahfghfh',
+  firstName: 'Jo',
+  lastName: 'DoeS',
 };
 
-export const user2 = {
-  UserName: 'kevinNew',
-  Email: 'mugishaisrael18@gmail.com',
-  Password: 'Password@2022',
-  FirstName: 'JohnS',
-  LastName: 'DoeS',
+export const invalidLastNameUser = {
+  userName: 'kevi',
+  email: 'itiswasp@gmail.com',
+  password: 'Andelahfghfh',
+  firstName: 'JohnS',
+  lastName: 'DS',
 };
-export const user3 = {
-  UserName: 'kevinNew',
-  Email: 'you@test.com',
-  Password: 'Password@2022',
-  FirstName: 'JohnS',
-  LastName: 'DoeS',
+
+export const dbInitUser = {
+  userName: 'kevinNew',
+  email: 'mugishaisrael18@gmail.com',
```

Comment:
If this is your real email, don't expose it in source codes.
Use envs. Do the same for everywhere real emails are exposed in source codes